### DISCUSSION TITLE
Files larger than 5mb gets pushed but does not reflect the changes on browser. The file I need to upload is 9.5 mb and the max size for Github is 25mb still i am facing this issue. - Github Desktop

### DISCUSSION THREAD

**Discussion by Abimomin** [community/community#134431] [Repositories]
### Select Topic Area

General

### GitHub Feature Area

GitHub Desktop

### Body

Hi Support,

I have a repository hosted on Github but when i upload files larget than 5mb it doesn't reflect the code on the browser. 

The file which i need to upload is 9.5 mb json file and the max size file limit for Github Desktop is 25mb.

Still i am facing the above issue. Need your help to resolve the issue.

Thanks.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Abimomin ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Abimomin** [reply]
Sorry for the inconvenience as this is my first post.

**Comment by marxwahlorian** [comment]
[]()

**Comment by marxwahlorian** [comment]
> []()

@marxwahlorian
